lib/hammer_plug.ex

```diff
@@ -210,7 +214,8 @@ defmodule Hammer.Plug do
       {:deny, _n} ->
         on_deny_handler.(conn, [])
 
-      {:error, _reason} ->
+      {:error, reason} ->
+        Logger.error(check_rate_fail_reason: reason)
```

Comment:
This should raise an exception, I think. Plug would catch it, send 500, and then re-raise.

Reply:
In Hammer as well, I think {:error, reason}` return value can be removed in favor of an exception since it's unclear what should be done in that case, allow or deny.

Reply:
Related: https://github.com/ExHammer/hammer-plug/pull/35